OSDOCS-11312: adds nit fixes to auto recovery MicroShift (#88145)
openshift/openshift-docs: modules/microshift-auto-recovery-example-rpm-systems.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-automation-example-rpm-systems_{context}"]
= Using auto-recovery in RPM systems

When {microshift-short} enters a failed state, the systemd service starts the `microshift-auto-recovery.service` unit. This unit executes the `auto-recovery` restore process.

> As a use case, consider the following example situation in which you want to automate the auto-recovery process for RPM systems that use the systemd service.

Reviewer: ```suggestion
As a use case, consider the following example situation in which you want to automate the automatic recovery process for RPM systems that use the systemd service.
```

.Procedure

> . Create a directory for the `microshift.service` ID by running the following command:

Reviewer 2: Ha, so in full sentence I'd say something different.
```suggestion
. Create a directory for the `microshift.service` systemd service unit by running the following command:
```

We could also drop `.service`:
```
. Create a directory for the `microshift` systemd service by running the following command:
```

$ sudo tee /usr/lib/systemd/system/microshift.service.d/10-auto-recovery.conf > /dev/null <<'EOF'
[Unit]
OnFailure=microshift-auto-recovery.service
StartLimitIntervalSec=25s # <1>
EOF
----

> <1> Increase the `StartLimitInterval` value from the default 10s to 25s for slower systems. A value too low can result in systemd never marking `microshift.service` as failed and the `OnFailure=` service not being triggered.

Reviewer 2: 25s was working for QE envs, but I wouldn't say it's an universal value.

Author: I was wondering that.

> <1> Increase the `StartLimitInterval` value from the default 10s to a larger value for slower systems. A value that is too low can result in systemd never marking the `microshift` systemd service as failed, which means that the `OnFailure=` service does not get triggered.

Reviewer: ```suggestion
<1> Increase the `StartLimitInterval` value from the default `10s` to a larger value for slower systems. A value that is too low can result in systemd never marking the `microshift` systemd service as failed, which means that the `OnFailure=` service does not get triggered.
```
